File: IT/Variabili, operatori, strutture di controllo/Esercizi/C++/esercizio18.cpp
/* file esercizio18.cpp */

#include <iostream>

using std::cout;
using std::endl;

int main()
{
	for (int i = 1; i <= 5; i++) {
		for (int j = 5; j >= 1; j--) {
			if (i == j) cout << i;
			else cout << " ";
		}
		cout << endl;
	}
}

File: IT/Variabili, operatori, strutture di controllo/Esercizi/Python/esercizio18.py
# file esercizio18.py

from __future__ import print_function
from sys import stdout
from os import linesep

for i in range(1, 6):
	for j in range(5, 0, -1):
		if i == j:
			print("{}".format(i), end="")
		else:
			stdout.write(" ")
	stdout.write(linesep) 

